flake.nix

```diff
@@ -0,0 +1,36 @@
+{
+  inputs = {
+    flake-utils.url = "github:numtide/flake-utils";
+    naersk.url = "github:nix-community/naersk";
+    nixpkgs.url = "github:NixOS/nixpkgs/nixpkgs-unstable";
+  };
+
+  outputs = { self, flake-utils, naersk, nixpkgs }:
+    flake-utils.lib.eachDefaultSystem (system:
+      let
+        pkgs = (import nixpkgs) {
+          inherit system;
+        };
+
+        naersk' = pkgs.callPackage naersk {};
```

Comment:
What is that? I'm not super familiar with Nix and used the naersk template suggested by their github repo.

Reply:
Overlays are a way to override packages definition (in a lazy recursive manner) but in that case they don't bring much. It would have let you extend the `pkgs` to include ones provided by naersk and have it available with `pkgs.naersk`.

In this case it doesn't matter and it's totally fine to stick to the official example.